backend/src/services/auth.js

```
@@ -14,335 +17,241 @@ import repo from "../repo/userRepository.js"; // Import the repository
-      // Create new user
-      const role = userData.role || "user";
-      console.log("📝 [AUTH SERVICE] Creating user in schema:", schema, "with role:", role);
-      
-      const result = await repo.createUser(schema, [
-         userData.name,
-         role,
-         userData.email,
-         userData.password, // In a real app, hash the password
-      ]);
+   const params = prepareInstance(instance, requiredFields);
+   const operation = service[operation];
```

Biome (1.9.4)

[error] Shouldn't redeclare 'operation'. Consider to delete it or rename it.

'operation' is defined here:


(lint/suspicious/noRedeclare)